diagnostic.md

```diff
@@ -73,14 +84,21 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request POST http://www.example.com/donuts/$ID /
```

Comment:
This will only work if `$ID` is set as an environment variable.  The header is also missing.

```sh
curl --include --request "POST" "http://example.com/donuts" \
  --header "Content-Type: application/json" \
  --data $'{
    "donut": {
      "name": "French Cruller",
      "price": "$0.99"
    }
  }'
```